parsl/configs/wqex_local.py

```diff
@@ -0,0 +1,14 @@
+from parsl.config import Config
+from parsl.executors import WorkQueueExecutor
+
+config = Config(
+    executors=[
+        WorkQueueExecutor(
+            label="wqex_local"
```

Comment:
CI is failing with a flake8 error here, which I think comes from a missing comma at the end of the line.

```
$ flake8 parsl/
parsl/configs/wqex_local.py:8:16: E999 SyntaxError: invalid syntax
```